components/timezone/src/ixdtf.rs

```
@@ -36,6 +34,8 @@ pub enum ParseError {
     InvalidOffsetError,
     /// There was an invalid IANA identifier provided.
     InvalidIanaIdentifier,
```

I don't think we ever want to raise `InvalidIanaIdentifier`, it might already be dead

---

Done